quadra/tasks/anomaly.py

```diff
@@ -386,10 +386,11 @@ def test(self) -> None:
                 batch_labels = batch_item["label"]
                 image_labels.extend(batch_labels.tolist())
                 image_paths.extend(batch_item["image_path"])
+                batch_images = batch_images.to(self.device).to(self.deployment_model.model_dtype)
```

Comment:
It should be possible to pass both the `device` and the `dtype` with single `.to()` call according to the documentation. https://pytorch.org/docs/2.1/generated/torch.Tensor.to.html#torch-tensor-to

Reply:
Crazy 🤯